politeiawww/convert.go

```diff
@@ -104,7 +103,6 @@ func convertPropCensorFromPD(f pd.CensorshipRecord) www.CensorshipRecord {
 
 func convertPropFromPD(p pd.ProposalRecord) www.ProposalRecord {
 	return www.ProposalRecord{
-		Name:             p.Name,
```

Comment:
Because the `Name` property is removed here, we need to ensure it's set on the `www.ProposalRecord` in the caller. From my testing, something like this works after https://github.com/decred/politeia/pull/141/files#diff-d9012ef76f0533a087446113b19c820eR1047:

    reply.Proposal.Name = cachedProposal.Name

Reply:
What about when it [loads](https://github.com/decred/politeia/blob/master/politeiawww/backend.go#L487) the remote inventory from pd?

If the caller has loaded inventory from that RPC call, then its `cachedProposal` wouldn't have `Name` either. Maybe to handle both cases, we can just set `Name` to the default string value (empty string) here in `convertPropFromPD`?

Reply:
That's a good point and explains some weird behavior I was seeing while testing. I think the only way to solve this will be to persist the proposal token & name in www. What do you think, @marcopeereboom?